tests/helpers/is-ie.js

```diff
@@ -0,0 +1 @@
+export default !window.ActiveXObject && 'ActiveXObject' in window;
```

Comment:
This is great! 👍  I think we should add a comment explaining what it does.

Something like:

```js
// `window.ActiveXObject` returns undefined in IE11 (as well as non-IE browsers)
// whereas `"ActiveXObject" in window` returns `true` in all IE versions
// only IE11 will pass _both_ of these conditions
```

Reply:
😨